crates/rpc/rpc-testing-util/src/trace.rs

```diff
@@ -97,6 +110,34 @@ pub trait TraceApiExt {
     fn trace_filter_stream<I>(&self, filters: I) -> TraceFilterStream<'_>
     where
         I: IntoIterator<Item = TraceFilter>;
+    /// Returns a new stream that yields the trace results for the given call requests.
+    fn trace_call_stream(
+        &self,
+        call: CallRequest,
+        trace_types: HashSet<TraceType>,
+        block_id: Option<BlockId>,
+        state_overrides: Option<StateOverride>,
+        block_overrides: Option<Box<BlockOverrides>>,
```

Comment:
this should be replaced with the new

https://github.com/paradigmxyz/reth/blob/e056bfb213e7ddb2f4345d9717735475229024dd/crates/rpc/rpc-types/src/eth/trace/tracerequest.rs#L12-L12

type which makes this a lot easier